concordium-cis2/src/lib.rs

```diff
@@ -836,7 +836,7 @@ impl schema::SchemaType for Receiver {
                 String::from("Contract"),
                 schema::Fields::Unnamed(vec![
                     ContractAddress::get_type(),
-                    OwnedReceiveName::get_type(),
+                    schema::Type::String(schema::SizeLength::U16),
```

Comment:
Perhaps a short comment what the String represents would be helpful in the future. It is also technically not necessary to have U16 for length. U8 should suffice for entrypoint names.

Reply:
I guess changing U16 -> U8 would break parsing, so never mind.

Reply:
added a comment